Makefile

```diff
@@ -230,5 +227,13 @@ gfx/unknown/unknown_egg.2bpp: rgbgfx += -h
 
 %.gbcpal: %.png
 	$(RGBGFX) -p $@ $<
+
 %.dimensions: %.png
 	tools/png_dimensions $< $@
+
+
+### These don't get built from other files, but have to exist on their own.
```

Comment:
There is little point in doing this. Make will fail if a required file isn't found, and not do anything to try to create them, unless a rule for them already exists, in which case this will override and clear said rule.

I propose that we should simply not define any rules for these files if we don't want make to make them ;)

Reply:
I assumed those rules existed for a reason, like to give better error messages. @yenatch added `pal` and `bin` in aaf26de8edab0c814463c24674c8a6f04c2a80f4, and `blk` and `tilemap` in 6122490cc339b5cab2ef9b25a7cf82b56de16413. ...Hmm, the `tilemap` rule is gone now, so maybe the rest can be deleted too?

Reply:
Extensions that don't get generated: `asm`, `bin`, `blk`, `pal`, `tilemap`, `tilemap.rle`, `attrmap`